src/componants/cart/type.js

```diff
@@ -1,20 +1,20 @@
-import bannerimg1 from '../../assets/bannerimg1.svg';
-import bannerimg2 from '../../assets/bannerimg2.svg';
+import bannerimg1 from "../../assets/bannerimg1.svg";
+import bannerimg2 from "../../assets/bannerimg2.svg";
 export var products = [
   {
     id: 1,
-    title: "Pastwords",
```

Comment:
It was supposed to be pastwords in visible strings as it is making fun of passwords. But if it is internal to code then it's ok to be passwords.